app/helpers/bookmarks_helper.rb

```diff
@@ -84,4 +84,52 @@ def get_count_for_bookmark_blurb(bookmarkable)
     content_tag(:span, link, class: "count")
   end
 
+  def bookmarker_id_for_css_classes(bookmark)
+    return if bookmark.nil?
+
+    "user-#{bookmark.pseud.user_id}"
+  end
+
+  # Array of unique creator and bookmarker ids, formatted user-123, user-126.
+  # If the user has bookmarked their own work, we don't need their id twice.
+  def user_ids_for_bookmark_blurb(bookmark)
+    user_ids = creator_ids_for_css_classes(bookmark.bookmarkable)
+    user_ids << bookmarker_id_for_css_classes(bookmark)
+    user_ids.uniq
+  end
+
+  # The bookmark blurb contains the bookmarkable and a single user's bookmark.
+  # If we cache this, it probably only needs to use the creation cache key.
+  def css_classes_for_bookmark_blurb(bookmark)
+    return if bookmark.nil?
+
+    creation_id = creation_id_for_css_classes(bookmark.bookmarkable)
+    user_ids = user_ids_for_bookmark_blurb(bookmark).join(" ")
+    "bookmark blurb group #{creation_id} #{user_ids}".strip
+  end
+
+  # I'm not using this yet.
+  # The bookmarkable blurb contains the bookmarkable and many users' bookmarks.
+  # This is equivalent to sticking the word "bookmark" in front of
+  # css_classes_for_creation_blurb(creation), but I don't know if we want to
+  # use the existing method because it's cached (the key is
+  # "#{creation.cache_key_with_version}/blurb_css_classes-v2").
```

Comment:
I like the approach you've done in the view file, where you're directly using `css_classes_for_creation_blurb`. I'm pretty neutral on whether to move that into a helper function, but I do think you should share the cache between this blurb and the creation blurb.

Reply:
It occurs to me that if the creation (bookmarkable) is nil, we're going to be left with just "bookmark" and not "bookmark blurb group" as the class. 

I don't think we intentionally display bookmarkable blurbs for deleted items anywhere (as opposed to bookmark blurbs, which are on the user's bookmarks page), _but_ I assume there's a possibility of them being visible a few moments while the index updates. 

So I think I might update `css_classes_for_creation_blurb` to return `blurb group` when the creation is nil.

Reply:
I think what happens is that the deleted bookmarkable is skipped and you get a header like "1 - 19 of 100 Bookmarked Items in Testing." So I don't think you can get `nil` bookmarkables in listings like that. But I haven't tested it recently.